hoot-core/src/main/cpp/hoot/core/io/HootApiDbReader.cpp

```diff
@@ -322,7 +322,7 @@ NodePtr HootApiDbReader::_resultToNode(const QSqlQuery& resultIterator, OsmMap&
       nodeId,
       resultIterator.value(ApiDb::NODES_LONGITUDE).toDouble(),
       resultIterator.value(ApiDb::NODES_LATITUDE).toDouble(),
-      ConfigOptions().getCircularErrorDefaultValue(),
+      -1,
```

Comment:
Yeah this was definitely part of the source of the problem.  Sometimes in some circumstances the `HootApiDbWriter` was writing the `error:circular` value out to the database when it wasn't found in the source data.  Then this would finish off the job and make all nodes contain that tag.